Source/APIClient/Request.swift

```diff
@@ -18,13 +18,15 @@ public enum HTTPMethod: String {
 public protocol Request {
     static var baseURL: String { get }
     static var version: String { get }
+    static var serverAddress: String? { get }
```

Comment:
After thinking about this a bit more, it would be a bit simpler to just expose the "Accept" header value entirely, instead of exposing the server and version separately and constructing the header value. This would also allow the user to specify a different format for the value, which will likely be necessary for non-Intrepid APIs.